roles/git_ssh_setup/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
We can get rid of this file as it is not required

Reply:
yep, done.